force-app/main/default/classes/DMLOperations.cls

```diff
@@ -31,13 +31,17 @@ public with sharing class DMLOperations {
      */
     public static Id insertNewAccount() {
         // Create a new Account instance
+        Account acc = new Account();
 
         // Set some fields on the Account
+        acc.Name = 'Random Test Account';
```

Comment:
Can you think of a way to create a random account name, such that each time this method is run a different name is generated?